index.md

```diff
@@ -178,6 +178,17 @@ instructions:
     - title: Community Supported
       summary: Packaged and supported by the wider community.
       distros:
+        - name: Alpine Linux
+          id: alpine
+          instructions: >-
+            Packages are available from the [community respositories](https://pkgs.alpinelinux.org/packages?name=openrazer).
+
+
+                doas apk add openrazer
```

Comment:
I think `openrazer-src` also needs to be installed explicitly?

Reply:
True, fixed.